src/Widgets/TableWidget/TableWidgetComponent.js

```diff
@@ -69,8 +69,7 @@ const AddMoreRows = Scrivito.connect(
       <tr>
         <th colSpan="4">
           <div className="text-center">
-            <a
-              href="#"
+            <span
```

Comment:
Now the "Click to add new row" does not indicate anymore, that it can be clicked (e.g. because the hover state is missing).